libs/types/src/cdf_cast_vote_records.ts

```diff
@@ -703,6 +705,17 @@ export interface CVR {
    * The sequence number for this CVR. This represents the ordinal number that this CVR was processed by the tabulating device.
    */
   readonly UniqueId?: string;
+
+  /**
+   * Indicates whether a ballot is absentee, precinct, or provisional
+   */
+  readonly vxBallotType: 'absentee' | 'provisional' | 'standard';
```

Comment:
Although I didn't actually import the type from the other file, I did just copy and paste this from existing `CastVoteRecords` type and it will make the conversion easier on the VxAdmin side if I just leave it as is for now.